src/style.css

```diff
@@ -45,3 +45,6 @@
 .playkit-floating-container.playkit-floating-active .playkit-floating-dismissible {
   display: inline-block;
 }
+.playkit-floating-dismissible.playkit-floating-infullscreen{
+  display: none !important;
```

Comment:
why do you need the !important ? is there a default display style ?

Reply:
@SivanA-Kaltura 
yes of:
```
.playkit-icon-close {
background-image: url(data:image/svg+xml,%3Csvg xmlns='[http://www.w3.org/2000/svg'](http://www.w3.org/2000/svg%27) xmlns:xlink='[http://www.w3.org/1999/xlink'](http://www.w3.org/1999/xlink%27) viewBox='0 0 1024 1024' width='36' height='36'%3E%3Cpath fill='%23fff' d='M573.162 512l214.269-214.269c16.772-16.772 16.688-44.071-0.202-60.96-17.007-17.007-44.182-16.98-60.96-0.202l-214.269 214.269-214.269-214.269c-16.772-16.772-44.071-16.688-60.96 0.202-17.007 17.007-16.98 44.182-0.202 60.96l214.269 214.269-214.269 214.269c-16.772 16.772-16.688 44.071 0.202 60.96 17.007 17.007 44.182 16.98 60.96 0.202l214.269-214.269 214.269 214.269c16.772 16.772 44.071 16.688 60.96-0.202 17.007-17.007 16.98-44.182 0.202-60.96l-214.269-214.269z' /%3E%3C/svg%3E);
}
```
so need the !important to override it.

Reply:
you can remove the icon with a condition in DismissibleFloatingButtonComponent